### ISSUE TITLE
Tests fail on FreeBSD with 0.56.0

### ISSUE THREAD

**Issue by ferki** [RexOps/Rex#515]
Example report: http://www.cpantesters.org/cpan/report/451243dc-8ff9-11e4-922e-9baedfbfc7aa

**Comment by ferki** [comment]
Looks like the tests themselves are passing, but there's a non-zero wait status at the end of running the tests:
```
t/xml.t .......................... 
All 12 subtests passed 

Test Summary Report
-------------------
t/xml.t                        (Wstat: 9 Tests: 12 Failed: 0)
  Non-zero wait status: 9
```

So it might be unrelated to `t/xml.t`, and rather be caused by a `Test::More` bug or similar.

**Comment by ferki** [comment]
According to CPAN Testers reports, it seems to be resolved since 1.3.0.
